Examples/CaseStudies/SwiftUICaseStudies/02-Effects-Basics.swift

```diff
@@ -52,7 +52,7 @@ let effectsBasicsReducer = Reducer<
     state.numberFact = nil
     // Return an effect that re-increments the count after 1 second.
     return .task {
-      try? await environment.mainQueue.sleep(for: 1)
+      try await environment.mainQueue.sleep(for: 1)
```

Comment:
can drop the `?` in a bunch of places since we allow cancellation errors to be thrown now.